tortoise/fixture_test.go

```diff
@@ -77,9 +77,11 @@ func (s *smesher) rawatx(id types.ATXID, publish types.EpochID, opts ...*aopt) *
 	return val
 }
 
+const publish = 1
```

Comment:
As stated before, epoch 1 is special different from all other epochs, we should have at least one test were we have a different publish epoch.